tools/destination-tester/README.md

```diff
@@ -145,9 +145,6 @@ Here is an example input file named `input_1.json`:
 
 The tester supports the following optional CLI arguments to alter its default behavior. You can append these options to the end of the `docker run` command provided in step 2 of [How To Run](https://github.com/fivetran/fivetran_sdk/tree/main/tools/destination-tester#how-to-run) section above.
 
-#### --port
```

Comment:
why are we removing this? Maybe change the description as --port is still a valid argument

Reply:
We have added --port in docker command, so removed it. Added back port cli with modified description